packages/swing-store/src/snapStore.js

```diff
@@ -231,12 +237,13 @@ export function makeSnapStore(
       if (fileStat) {
         return freeze(info);
```

Comment:
Since we `stat`ed the file, should we include the `compressedByteCount` with a `0` or `undefined` `compressSeconds` ?

Reply:
Good point... I think we do want to emit fields consistently, including data types. I've opted to read `compressedByteCount` from stat info, use 0 for `compressSeconds`, and introduce a boolean `newFile` to support exclusion of those values for accurate statistics.